src/test/java/com/fasterxml/jackson/core/BaseTest.java

```diff
@@ -451,11 +451,15 @@ protected String readAndWrite(JsonFactory f, JsonParser p) throws IOException
         g.disable(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT);
         try {
             while (p.nextToken() != null) {
+                System.err.println(p.currentToken() + "  " + p.currentName() + "  " + p.currentValue());
```

Comment:
Uh. Should have noticed, will remove after merge.

Reply:
Woops! I hadn't noticed that I left that open either.

In Elasticsearch we use https://github.com/policeman-tools/forbidden-apis to prevent these kinds of things sneaking in. I still use them for debugging frequently enough, but I can't get the build to pass with them.